index.html

```diff
@@ -138,7 +139,9 @@ <h6>1,000 views | 1 year ago</h6>
             </div>
           
         </div>
+        </section>
 
+        <section>
         <div id="video-container">
             <div class="video-template">
                 <img class="video-image" src="https://picsum.photos/400/200" alt="">
```

Comment:
## Missing image alt tag
Please make sure to include alt tag to all images as this improves SEO and its good practice for good code.